### ISSUE TITLE
The "seo" cache configuration does not exist.

### ISSUE THREAD

**Issue by akkaweb** [orgasmicnightmare/cakephp-seo#1]
It has been hard to get this plugin up and running. Perhaps it is not fully developed yet. I have done a few things to get it up. With the nonexistence of default templates and no instructions or mentions it makes it difficult. Also quick code snippets on how to add this into an app, would be helpful, mainly behavior and component. Although I believe I have done most of the things to get it up, I am having an error that I have not been able to fix:

```
The "seo" cache configuration does not exist.
```

I have looked and I dont seem to see what Configuration it expects. Can you help out with this?

Thanks,

**Comment by akkaweb** [comment]
After some more searching into CakePHP docs, I was able to add:

```
'Cache' => [
        'seo' => [
            'className' => 'File',
            'prefix' => 'myapp_seo',
            'path' => CACHE . 'persistent/',
            'serialize' => true,
            'duration' => '+2 days',
        ]
    ]
```

**Comment by osazos** [comment]
Yep the plugin is still under development but you right a better documentation should be exists. I'll try to do it when I have the time.
